pages/common/rustfmt.md

```diff
@@ -0,0 +1,15 @@
+# rustfmt
+
+> Tool for formatting Rust source code.
+
+- Format a file, overwriting the original file in-place:
+
+`rustfmt {{source.rs}}`
+
+- Check a file for formatting and display any changes on the console:
+
+`rustfmt --check {{source.rs}}`
+
+- Backup any modified files before formatting (if a file `source.rs` is formated, the backup is `source.bk`):
```

Comment:
How about we simplify it to - `(the original file is renamed with a '.bk' extension)`